code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -2781,15 +2781,15 @@
 	flavor = newflavor
 
 	switch(newflavor)
-		if("boneless pork ribs", "grilled chicken", "pizza square", "spaghetti chunks", "chicken tender")
+		if("boneless pork ribs", "grilled chicken", "pizza square", "spaghetti", "red crayon")
 			icon_state = "entree"
 			list_reagents = list("nutriment" = 5, "sodiumchloride" = 1)
-		if("cracker", "cheese spread", "rice onigiri", "mashed potatoes", "risotto")
+		if("meatballs", "cheese spread", "beef turnover", "mashed potatoes", "yellow crayon" )
 			icon_state = "side"
 			list_reagents = list("nutriment" = 3, "sodiumchloride" = 1)
-		if("biscuit", "meatballs", "pretzels", "peanuts", "sushi")
+		if("biscuit", "pretzels", "peanuts", "cracker", "purple crayon")
 			icon_state = "snack"
 			list_reagents = list("nutriment" = 2, "sodiumchloride" = 1)
-		if("spiced apples", "chocolate brownie", "sugar cookie", "coco bar", "flan", "honey flan")
+		if("spiced apples", "chocolate brownie", "sugar cookie", "choco bar", "blue crayon")
```

Comment:
I see you removed the chicken tender, risotto, sushi, flan and honey flan, was it intended?

Reply:
Yes